packages/lit-element/src/lit-element.ts

```diff
@@ -54,13 +54,18 @@
  *
  * @packageDocumentation
  */
-import {PropertyValues, UpdatingElement} from './updating-element.js';
+import {PropertyValues, UpdatingElement} from './lib/updating-element.js';
 import {render, RenderOptions} from 'lit-html';
-import {supportsAdoptingStyleSheets, CSSResult, unsafeCSS} from './css-tag.js';
+import {supportsAdoptingStyleSheets, CSSResult, unsafeCSS} from './lib/css-tag.js';
 
-export * from './updating-element.js';
+export * from './lib/updating-element.js';
 export {html, svg, TemplateResult} from 'lit-html';
-export * from './css-tag.js';
+export * from './lib/css-tag.js';
+
+const DEV_MODE = true;
+if (DEV_MODE) {
+  console.log('lit-element is in dev mode. Not recommended for production!')
```

Comment:
One thing I noticed with this is that it prints during tests for every test file we have, which is just a bit of an annoyance. I wonder if we could skip this message during tests?

Reply:
We could add a rollup plugin that suppresses the log statements I guess? Punting for now.